auction-server/src/auction/repository/add_bid_analytics.rs

```diff
@@ -68,13 +68,13 @@ impl Repository {
             entities::BidTransactionData::Swap(transaction_data) => {
                 let status_reason = Svm::get_bid_status_reason(&bid.status);
                 let mint_user = transaction_data.accounts.mint_user;
-                let user_token_usd_price = calculate_final_amount(
+                let user_token_usd_price = calculate_notional_value(
```

Comment:
this variable should be named `user_token_notional_value` instead of `user_token_usd_price`

Reply:
or you can leave it as price and just not calculate the notional value yourself. instead you can literally just store the decimals-adjusted price here with a bunch of precision

Reply:
otherwise, the column name in the db `buy_token_usd_price` is very misleading. this refers to the USD price of the token, not the notional value of the total amount of the tokens